error.go

```diff
@@ -130,6 +131,34 @@ func IsPullChartNotFound(err error) bool {
 	return microerror.Cause(err) == pullChartNotFoundError
 }
 
+var pullChartTimeoutError = &microerror.Error{
+	Kind: "pullChartTimeoutError",
+}
+
+// IsPullChartTimeout asserts pullChartTimeoutError.
+func IsPullChartTimeout(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	c := microerror.Cause(err)
+
+	if c == pullChartTimeoutError {
+		return true
+	}
+
+	netErr, ok := err.(net.Error)
+	if !ok {
+		return false
+	}
+
+	if netErr.Timeout() {
```

Comment:
What about `return netErr.Timeout()`? since it's bool returning anyway.

Reply:
Nice. Done.